### PR TITLE
assertions scoreData

### PR DESCRIPTION
**Pull request by grewered**

### CODE REVIEW

File: R/scoreData.r [hunk 1]
```diff
@@ -8,6 +8,9 @@ scoreData <- function(
     verbose = FALSE,
     ... # for the deprecated units argument
 ) {
+  lapply(c(unitrecodings, subunits), checkmate::assert_vector)
```
Comment (grewered) [inline R/scoreData.r:11]: I wasn't sure whether the arguments have to have a specific type

Comment (sachseka) [reply R/scoreData.r:11]: both are data.frames

**Review by sachseka — CHANGES_REQUESTED** [review]

**Comment by sachseka** [comment]
Please pay attention to checks

**Comment by grewered** [comment]
grafik
here it's the same as in #25, I checked for data frames, but apparently the arguments are character

**Comment by sachseka** [comment]
> grafik here it's the same as in #25, I checked for data frames, but apparently the arguments are character. I checked the test-scoreData.r file, and the test objects for `unitrecodings` and `subunits` are both data frames. So I don't know where the error comes from.

This comes from concetanating the data.frame with "c(dat1, dat2, ...)" instead of "list(dat1, dat2, ...)" Please change this accordingly

File: R/scoreData.r [hunk 2]
```diff
@@ -8,6 +8,9 @@ scoreData <- function(
     verbose = FALSE,
     ... # for the deprecated units argument
 ) {
+  lapply(c(unitrecodings, subunits), checkmate::assert_data_frame)
```
Comment (sachseka) [inline R/scoreData.r:11]: please use list() instead of c()

**Review by sachseka — CHANGES_REQUESTED** [review]

**Pull request merged**